twoliter/src/cmd/build.rs

```diff
@@ -57,7 +57,7 @@ impl BuildVariant {
         install_tools(&toolsdir).await?;
         let makefile_path = toolsdir.join("Makefile.toml");
         // A temporary directory in the `build` directory
-        let build_temp_dir = TempDir::new_in(project.project_dir())
+        let build_temp_dir = TempDir::new_in("/tmp")
```

Comment:
/tmp can be a memory-backed tmpfs so it is unwise to leak files there - and can be pretty annoying to clean up.

Let's use a path like build/.tmp that we recursively delete and create fresh.

Reply:
> Let's use a path like build/.tmp that we recursively delete and create fresh.

Note that an `rm -rf "${BUILDSYS_BUILD_DR}/.tmp"` needs to be added to `Makefile.toml` in the appropriate `clean` task if we do this here. Also, a `create_dir_all` is probably required here for the `build` directory because we are doing these things before we go into `cargo make`.